frontend/src/static/js/api.js

```diff
@@ -93,4 +101,26 @@ window.api = {
         });
   },
 
+  loadAuthorCommitsVariance() {
```

Comment:
Just to check, which/where is the repo's variant for this function?

Reply:
It's inside `loadCommits`, lines 87-90

Reply:
Doesn't seems like a clean solution. E.g. the repo and author variant of calculating commits and variance are divided into 2 functions with diff name (`loadCommits` and `loadAuthorCommitsVariance`) and different variable names e.g. `User` vs `Author` which increase the amount of time needed to understand the code.

Perhaps try to extract it as a common method? E.g. `calcCommitsVariance(repos)`